### PR TITLE
Tickets/dm 35547

### PR DESCRIPTION
**Pull request by teweitsai**
* Use the [ts_m2com](https://github.com/lsst-ts/ts_m2com).
* Fix the indentation of *version_history.rst*.

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+v0.5.6
```
Comment (tribeiro) [inline doc/version_history.rst:5]: I think this is a quite substantial change. Consider at least increasing the minor revision, `v0.6.0`

File: python/lsst/ts/m2/csc.py [hunk 2]
```diff
@@ -129,7 +130,9 @@ def __init__(
         self._port_telemetry = port_telemetry
```
Comment (tribeiro) [inline python/lsst/ts/m2/csc.py:130]: Somewhere here in the init file, consider setting the version of ts-m2comm in the `softwareVersions` event.

```
from lsst.ts.m2comm import __version__ as __m2comm_version__
```

```
self.evt_softwareVersions.set(subsystemVersions=f"ts-m2comm={__m2comm_version__}")
```

**Review by tribeiro — APPROVED** [review]
looks good, see my comments inline though.

File: doc/version_history.rst (hunk 1 above)
Comment (teweitsai) [reply doc/version_history.rst]: Done in the commit: [2ce50af](https://github.com/lsst-ts/ts_m2/pull/28/commits/2ce50af7b174ca88c63068bb94e185ac8586ddb9). Thanks!

File: python/lsst/ts/m2/csc.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py:131]: Done in the commit: [2ce50af](https://github.com/lsst-ts/ts_m2/pull/28/commits/2ce50af7b174ca88c63068bb94e185ac8586ddb9). Thanks!

**Pull request merged**